code/modules/antagonists/pirate/pirate_roles.dm

```diff
@@ -94,3 +94,37 @@
 
 /obj/effect/mob_spawn/ghost_role/human/pirate/silverscale/gunner
 	rank = "Top-drawer"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne
+	name = "\improper Interdyne sleeper"
+	desc = "A cryo sleeper, looking impeccably clean!"
+	density = FALSE
+	you_are_text = "You are an Ex-Interdyne pharmacyst now turned space pirate."
+	flavour_text = "The station has refused to fund your research, you will now have to convince them to donate to your charitable cause."
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	prompt_name = "an Ex-Interdyne employee"
+	outfit = /datum/outfit/pirate/interdyne
+	rank = "Pharmacist"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne/senior
+	rank = "Head Pharmacist"
+	outfit = /datum/outfit/pirate/interdyne/captain
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/interdyne/junior
+	rank = "Pharmacist"
+
+/obj/effect/mob_spawn/ghost_role/human/pirate/grey
+	name = "\improper Assistant sleeper"
+	desc = "A cryo sleeper, it looks kind of dirty"
+	density = FALSE
+	you_are_text = "You used to be an Assitant contracted by nanotrasen but their rules shackled your potential now you are truly FREE."
+	flavour_text = "You know you can beat the whole station, and you will steal everything they hold dear."
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	prompt_name = "An assitant gone loose"
```

Comment:
```suggestion
	prompt_name = "A rogue assistant pirate"
```

Reply:
I like more An assitant gone loose since it highlights their past a rogue assistant pirate sounds like they are just the assistant of piracy and not a previous assitant